changelogs/fragments/snmp_facts.yml

```
@@ -0,0 +1,2 @@
+security_fixes:
+- **SECURITY** - CVE-2021-20178 - hide user sensitive information such as ``privkey`` and ``authkey`` from logging into the console.
```

```suggestion
- snmp_facts - **CVE-2021-20178** - hide user sensitive information such as ``privkey`` and ``authkey`` from logging into the console (https://github.com/ansible-collections/community.general/pull/1621).
```
It's already in a section "Security Fixes", so the extra `**SECURITY**` should not be needed.